docker-compose.yml

```diff
@@ -25,8 +25,6 @@ services:
       POSTGRES_USER: ${POSTGRES_USER-postgres}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD-postgres}
     volumes:
-      - db_data:/var/lib/postgresql/data
+      - ~/rootstrap/projects/databases/action-mailbox:/var/lib/postgresql/data
```

Comment:
🤔 this path seems a little bit custom, maybe something more generic like the previous one?

Reply:
done @megatux